### DISCUSSION TITLE
[FeatureRequest] Hide datalinks when there is no data in the target URL

### DISCUSSION THREAD

**Discussion by hoyeunglam** [grafana/grafana#63925] [Dashboards, Panels & Visualizations]
**What would you like to be added:**

A feature to hide datalinks without data in the target url. 
For example if there's a geomap with specific markers it would be favorable to hide the links to other dashboards without data in it.

![Datalink](https://user-images.githubusercontent.com/31922701/222118595-2e3251a0-3417-450d-a07e-e3b964f33177.png)

**Why is this needed:** 

This feature would improve overall visibility of data for the end users. 
Since the datalinks are currently static the datalinks are showing up on every marker without checking if there's any data in the targeted URL. This results into users being redirected to dashboards without data.

![image](https://user-images.githubusercontent.com/31922701/222121205-2bb86065-e4a8-40dd-b75e-fafe7c04238e.png)

Thank you!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82805. Feel free to continue the discussion around this there. Thank you!
